examples/chef/chef.py

```diff
@@ -521,6 +521,12 @@ def main() -> int:
     # Cluster customization
     #
 
+    if options.do_run_gui or options.do_run_zap:
+        if (shell.run_cmd(f"if [[ -x $(which zap-cli) ]]; then echo \"zap-cli found\"; else echo \"\"; fi", return_cmd_output=True) != "zap-cli found\n"):
```

Comment:
And this part should probably be fixed to work with a source ZAP, not just a prebuilt one.

Reply:
we have a zap_execution.py that was intended to give a central point that handles environment variables as well as printing out verbose errors when zap-cli is not found.